### ISSUE TITLE
Tooltip not accessible

### ISSUE THREAD

**Issue by dialpad-drew** [dialpad/dialtone#203]
The Dialtone Tooltip example mark-up isn't accessible as it does not announce to voiceover and it's only visible on `:hover`. Can we use the Vue component as inspiration (whenever it's completed) to update the example mark-up in the documentation?
